apps/src/templates/instructions/DynamicInstructions.jsx

```diff
@@ -87,7 +88,19 @@ class DynamicInstructions extends React.Component {
                 ref={ref => (this.dynamicInstructionsRefs[key] = ref)}
               >
                 <div style={{fontSize: 16}}>
-                  {this.props.dynamicInstructions[key]}
+                  <div style={{top: 10, position: 'absolute'}}>
+                    <Typist
+                      avgTypingDelay={35}
+                      stdTypingDelay={15}
+                      cursor={{show: false}}
```

Comment:
Can we pull the in-line styles into a constant at the bottom of the file and set the magic numbers to variables, please?